infor-design/enterprise
# 2722 - Q/A issues for Fontpicker component [v4.24.x]

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR addresses these two issues:
- Changes to settings handling on Toolbar Flex to accommodate passing down Popupmenu settings to an action button caused some AJAX calls not to be properly handled.  The logic for passing settings has been changed to correctly target action buttons.
- Fixed the alignment of the dropdown icon on both Edge (Vibrant) and Firefox (Subtle)

**Related github/jira issue (required)**:
Closes #2722

**Steps necessary to review your pull request (required)**:
Pull this branch, build, run the app.  Then...

_To test the Ajax Calls are working properly:_
- Open http://localhost:4000/components/toolbar-flex/example-more-actions-ajax.html?layout=nofrills
- Open the "Menu Button".  All of its child items should be prefixed with "Menu Button Option".
- Open the More Actions menu.  All of its child items should be prefixed with "AJAX Option"

_To test the icon alignment:_
- Open http://localhost:4000/components/editor/example-flex-toolbar in Firefox.  The general button icon alignment compared to the Fontpicker's should be accurate.
- Open http://localhost:4000/components/editor/example-flex-toolbar?theme=uplift in IE/Edge.  The same should be true in these browsers.
